en/config_fw/weight_and_altitude_tuning.md

```diff
@@ -0,0 +1,127 @@
+# Advanced TECS Tuning (Weight and Altitude)
+
+Before following the guidance provided by this section you should have completed the [basic TECS tuning](../config_fw/position_tuning_guide_fixedwing.md#tecs-tuning-altitude-and-airspeed) which established the key performance limitations of the vehicle that are very important for the altitude and airspeed controller to function properly.
+While those limitations have been defined using constant parameters, in reality vehicle performance is not constant and is affected by various factors.
+This section specifically looks at how vehicle weight and air density affect performance and what you can do do compensate for those effects.
+
+Both the vehicle weight and the air density (depends on altitude and air temperature) affect vehicle performance.
+For example, when a vehicle climbs to a higher altitude the air density decreases and usually the maximum climb rate reduces.
+Similarly, the maximum climb rate of a vehicle will reduce with the weight of the vehicle.
+
+If changes in weight and air density are not taken into account, altitude and airspeed tracking will likely deteriorate in the case where the configuration (air density and weight) deviate significantly from the configuration at which the vehicle was tuned.
+
+In the following we will use the notation $\hat X$ to specify that this value is a calibrated value of the variable $X$.
+By calibrated we mean the value of that variable measured at sea level in standard atmospheric conditions and when vehicle weight was equal to [WEIGHT_BASE](../advanced_config/parameter_reference.md#WEIGHT_BASE).
+
+E.g. by $\hat{\dot{h}}_{max}$ we specify the maximum climb rate the vehicle can achieve at [WEIGHT_BASE](../advanced_config/parameter_reference.md#WEIGHT_BASE) at sea level in standard atmospheric conditions.
+
+## Compensating for Weight
+
+[WEIGHT_BASE](../advanced_config/parameter_reference.md#WEIGHT_BASE) specifies the base weight of the vehicle at which the [TECS tuning](position_tuning_guide_fixedwing.md#tecs-tuning-altitude-and-airspeed) was performed.
+The weight of the vehicle in the tuning configuration should be measured with a scale and the parameter should be set accordingly.
+
+[WEIGHT_GROSS](../advanced_config/parameter_reference.md#WEIGHT_BASE) specifies the actual weight of the vehicle at any given time.
+This parameter can be set to reflect e.g. a payload attached to the vehicle which was not present during the tuning phase.
+
+When both [WEIGHT_BASE](../advanced_config/parameter_reference.md#WEIGHT_BASE) and [WEIGHT_GROSS](../advanced_config/parameter_reference.md#WEIGHT_GROSS) are set to a number larger than 0 PX4 will use this information to:
+
+### Scale the Maximum Climb Rate
```

Comment:
@RomanBapst I've just run prettier over this and removed the links from the headings. The text from the heading is now in first sentence. Generally better not to have links in section headings.